docs/docsite/rst/user_guide/playbooks_delegation.rst

```diff
@@ -278,6 +278,9 @@ As always with delegation, the action will be executed on the delegated host, bu
 .. note::
     Any conditional (i.e `when:`) will use the variables of the 'first host' to decide if the task runs or not, no other hosts will be tested.
 
+.. note::
+    If you want to avoid the default behaviour of setting the fact for all hosts `delegate_facts: True` also needs to be specified for the specific task or block.
```

Comment:
```suggestion
    If you want to avoid the default behaviour of setting the fact for all hosts, set `delegate_facts: True` for the specific task or block.
```